tests/integration/utils.py

```
@@ -0,0 +1,24 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Any
+
+from argilla.client.api import delete
+
+
+def delete_ignoring_errors(*args: Any, **kwargs: Any) -> None:
```

What is this doing exactlyy, shouldn't we specific the `Exception` a bit better and pass a docstring about the usage?

---

this is doing what the old `delete` was doing, which was to not return any exception. We were extensively using the `delete` as the first step in our unit tests, to remove datasets that could have beee not removed from previous unit tests executions. As `delete` now raises an exception, we cannot use it directly, and I had to create this function which tries to delete the dataset and if it doesn't exist, then just ignores the exception.